### DISCUSSION TITLE
Debug in IntelliJ always leads to "Failed to start gauge API: Timed out connecting to java"

### DISCUSSION THREAD

**Discussion by lochotzke** [getgauge/gauge#1852] [General]
Hello,

I'm able to execute a single spec just fine. Whenever I try to start the step in debug mode in IntelliJ I get "Failed to start gauge API: Timed out connecting to java".

My current gauge version:
windows, 1.1.6, 3af456f
html-report (4.0.12), java (0.7.13), screenshot (0.0.1), xml-report (0.2.3)

I haven't tried 1.1.7 yet since colleagues of mine can use debugging.

I already increased my timeout a lot but that didn't help:
ide_request_timeout = 300000
plugin_connection_timeout = 100000
plugin_kill_timeout = 40000
runner_connection_timeout = 300000
runner_request_timeout = 300000

Do you have any idea what the issue might be?

Regards
André

**Comment by sriv** [comment]
hi, do you see any errors in the logs? Check your project's log directory for `gauge.log` or the `.out` files, and these might be able to give some hints.

**Comment by vlebedenko** [comment]
Hi, 
I get the same error. Only in Debug mode.
Firstly I get debug error :
`Error running '': Unable to open debugger port (localhost:60677): java.net.ConnectException "Connection refused: connect"`

and then in console
```
Runner Ready for Debugging
Listening for transport dt_socket at address: 60677
Error ----------------------------------

[Gauge]
Failed to start gauge API: Timed out connecting to java
```
I've tried to change different params in Configuration, like arguments - address=5005 and a lot of other, but nothing helped me. Error always shows random port. 
Haven't any .out logs or gauge.log in project.
Is there any way to fix it ?

Environment:
windows, 1.4.3, f98dd40
	html-report (4.2.0), java (0.10.2), screenshot (0.1.0), spectacle (0.1.4)

**Reply by antondebeer** [reply]
Did you manage to get this sorted?

**Reply by vlebedenko** [reply]
> Did you manage to get this sorted?

Yes. I've changed my job 😄

**Comment by antondebeer** [comment]
No solution for this issue???
